FeedStoreChallenge/CoreData/Entities/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Антон on 5.09.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
+	@NSManaged var timestamp: Date
+	@NSManaged var feed: NSOrderedSet
+
+	var localFeed: [LocalFeedImage] {
+		return feed.compactMap { ($0 as? ManagedFeedImage)?.local }
+	}
+}
+
+extension ManagedCache {
+	static func find(in context: NSManagedObjectContext) throws -> ManagedCache? {
+		let request = NSFetchRequest<ManagedCache>(entityName: String(describing: self))
```

Comment:
Class name and entity name can be different, they are not required to be the same. Thus using a class name (a string describing the class) as an entity is not correct. To get the entity name you can use `entity().name!`
```suggestion
		let request = NSFetchRequest<ManagedCache>(entityName: entity().name!)
```